docs/docs/getting-started/02-oauth-tutorial.mdx

```diff
@@ -85,22 +101,22 @@ export default function App({
 }
 ```
 
-Instances of `useSession` (more on it in the next section) will then have access to the session data and status. The `<SessionProvider />` also takes care of keeping the session updated and synced between browser tabs and windows. 💪🏽
+Instances of `useSession` (more on it in the next section) will have access to the session data and status. The `<SessionProvider />` also keep the session updated and synced between browser tabs and windows. 💪🏽
```

Comment:
Move this up to satisfy https://github.com/nextauthjs/next-auth/pull/6551/files#r1089949056